src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -26,6 +27,7 @@ interface IBlazor {
   start?: ((userOptions?: Partial<CircuitStartOptions>) => Promise<void>) | ((options?: Partial<WebAssemblyStartOptions>) => Promise<void>);
   platform?: Platform;
   rootComponents: typeof RootComponentsFunctions;
+  webAssemblyLoadingSetProgress?: SetProgressFunction;
```

Comment:
@SteveSandersonMS @javiercn this is a public API change. Just wanted to confirm if you had any thoughts or concerns about it. 

Users would be able to provide their own set progress function here for custom progress reporter implementations. We'd be calling this function by providing the number of loaded assets so far and number of total assets.